project1/templates/book.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<html class="no-js">
+
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Use a generic template and extend this file to it.

Reply:
Layout added and extended